### PR TITLE
Update e2e tests for the new pages

### PR DESCRIPTION
**Pull request by torhector2**
E2E tests didn't pass due to some changes on the app's pages. This PR updates the tests to make the tests pass. 

I increased the wait time a little bit because when I ran the tests, the network was busy and we were having some timeouts (only in the DSR spec).

We could add more functionality to the tests if needed (maybe mock the delegates request to make it response always the same number of results), but my purpose with this PR was to set the test up and running again.

### CODE REVIEW

File: apps/webapp/tests/e2e/specs/delegates.spec.ts [hunk 1]
```diff
@@ -4,12 +4,12 @@ describe('Delegates Page', () => {
       // eslint-disable-next-line cypress/no-unnecessary-waiting
       cy.wait(1000).then(() => {
         cy.get('[data-testid="delegate-card"]').should('be.visible');
-        cy.get('[data-testid="delegate-card"]').should('have.length', 10);
+        cy.get('[data-testid="delegate-card"]').should('have.length', 27);
```
Comment (rafinskipg) [inline apps/webapp/tests/e2e/specs/delegates.spec.ts:7]: it shouldn't be like that. Delegates should load 10 by 10, do not load 27 and then 54. If that is happening, something is wrong

**Review by rafinskipg — CHANGES_REQUESTED** [review]

File: apps/webapp/tests/e2e/specs/delegates.spec.ts (hunk 1 above)
Comment (adamgoth) [reply apps/webapp/tests/e2e/specs/delegates.spec.ts:7]: I changed it to 27 so this is actually correct

**Review by adamgoth — APPROVED** [review]
Thanks for fixing these

File: apps/webapp/tests/e2e/specs/delegates.spec.ts (hunk 1 above)
Comment (torhector2) [reply apps/webapp/tests/e2e/specs/delegates.spec.ts:7]: Merging the branch as it seems to be the correct behaviour. Merging by checking the "Merge without waiting for requirements to be met (bypass branch protections)" option.

**Pull request merged**